x-pack/legacy/plugins/maps/public/components/_index.scss

```diff
@@ -2,6 +2,26 @@
   margin-bottom: $euiSizeS;
 }
 
+.mapMetricEditorPanel__metricEditor {
```

Comment:
@cchaos I noticed that all the custom CSS is inside the `_index.scss`. Should I move them to their own files and then import on `_index.scss`?

Reply:
It **should** be moved to its own file. But in the interest of time, I'm going to approve this PR as is and you can follow up with a new branch that maybe turns on linting and fixes up the file structures?

Reply:
Yes, no problem!